### ISSUE TITLE
Error using Ctrl+G to find vars

### ISSUE THREAD

**Issue by GreatScott000** [wbrakowski/AL-Navigator#45]
When I use Ctrl+G in a Page it works fine.  However when I do it in a Codeunit I get and error.  If it matters this is a Test Codeunit.

> Running the contributed command: 'alvarhelper.GlobalVars' failed.

See attached image.
![image](https://user-images.githubusercontent.com/4333956/100337644-920d1a00-2fcf-11eb-91bb-3a83730ad7ee.png)

**Comment by wbrakowski** [comment]
Hey GreatScott000,
thanks for reporting the issue. As the error message says the command "alvarhelper.GlobalVars" failed. 
The command belongs to the extension AL Variable Helper.

Here is a list of the commands for variables provided by AL Variable Helper and AL Navigator.
The first two belong to my extension, the other two belong to AL Variable Helper.

![image](https://user-images.githubusercontent.com/53928639/100342515-5f1e5280-2fde-11eb-9195-442b3c93a1d8.png)

Long story short: Please report this issue in [Rasmus' github](https://github.com/RasmusTidselbak/al-var-helper/issues).

**Comment by GreatScott000** [comment]
Hi Sorry about that, I obviously clicked on the wrong extension..
